app/src/main/res/navigation/main.xml

```diff
@@ -17,7 +18,8 @@
     <fragment
         android:id="@+id/sections_fragment"
         android:name="guide.graphql.toc.ui.sections.SectionsFragment"
-        tools:layout="@layout/sections_fragment">
+        tools:layout="@layout/sections_fragment"
+        android:label="{chapterTitle}">
 
```

Comment:
Here we set the label of the fragment to the chapterTitle argument that we pass in when navigating